.appveyor.yml

```diff
@@ -149,6 +149,9 @@ build_script:
   - ps: >
       cp C:\projects\mlpack\src\mlpack\tests\data\german.csv
       C:\projects\mlpack\dist\win-installer\staging\examples\sample-ml-app\sample-ml-app\data\
+  - >
+    "%MSBUILD%" "C:\projects\mlpack\dist\win-installer\staging\examples\sample-ml-app\sample-ml-app.sln"
+    /m /verbosity:minimal /nologo /p:BuildInParallel=true
```

Comment:
Should we be using the `.appveyor/sample-ml-app.vcxproj` file here that you added?

Reply:
I was using the file in `doc/examples` because the directory was already copied over into the staging directory. I will try switching directories and see if that changes anything

Reply:
Yeah, it just seemed to me like the `.appveyor/` vcxproj is different, so that is probably the one we should use here.  (I suppose it's possible we may need to make one for each version of Visual Studio that AppVeyor builds for? I am not sure).

Reply:
It still throws the same error of the Windows SDK not being found. As for separate scripts, I 10.0.18362 seems to be pre-installed on both 2015 and 2017 images of Visual Studio, so I think that being a problem is unlikely.